sed/tests/test.bats

```diff
@@ -0,0 +1,20 @@
+source "${BATS_TEST_DIRNAME}/../results/last_build.env"
+
+@test "sed binary exists" {
+ [ -f "/hab/pkgs/$pkg_ident/bin/sed" ]
+}
+
+@test "Help command" {
+  run hab pkg exec core/sed sed --help
+  [ $status -eq 0 ]
+  [[ "$output" =~ Usage:.+$pkg_ident/bin/sed ]]
```

Comment:
Ideally don't use `[[ .. ]]` in BATS. We don't need to match the output of the help command, just the exit code is really enough. Thus we can avoid `[[ .. ]]` usage.